client/apps/edit/components/admin/article/index.coffee

```diff
@@ -167,6 +171,8 @@ module.exports = AdminArticle = React.createClass
               input {
                 type: 'checkbox'
                 checked: @props.article.get 'exclude_google_news'
+                value: @props.article.get 'exclude_google_news'
+                readOnly: true
```

Comment:
React was surfacing errors about values set on uncontrolled components (because the onChange event for our styled checkbox is in the parent div).